hack/preview.sh

```diff
@@ -86,3 +97,26 @@ git checkout $MY_GIT_BRANCH
 
 #set the local cluster to point to the current git repo and branch and update the path to development
 $ROOT/hack/util-update-app-of-apps.sh $MY_GIT_REPO_URL development $PREVIEW_BRANCH
+
+# Make sure we have a tekton-chains namespace
+echo "Checking to see if tekton-chains namespace exists"
+while ! kubectl get namespace tekton-chains &> /dev/null; do
+  echo -n .
+  sleep 3
+done
+
+echo "Setting chains to use cluster rekor server: https://rekor.$domain"
+kubectl patch configmap/chains-config -n tekton-chains --patch "{\"data\":{\"transparency.url\":\"https://rekor.$domain\"}}" --type=merge 
```

Comment:
Maybe we can use the config.sh here, e.g.:

      echo "Setting ..."
      $ROOT/hack/chains/config.sh rekor-local

It would also take care of the pod restart.

Reply:
Just an idea, maybe not for this PR, you could do a sed ... commit -m on `components/build/tekton-chains/chains-config.yaml` and then Gitops would apply the chains-config change as well.